cabal.project

```diff
@@ -217,8 +217,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: e9cda57df7ea6969edbc3bfc4e117668277d09c8
-  --sha256: 1a91z6yvlhzlqrf7fy51fmmpm3ssk684h6pjgg022cdlsq56yif2
+  tag: f36b37168b3790c5e3a1974994b04de188d8b0ae
```

Comment:
This revision needs to be https://github.com/input-output-hk/ouroboros-network/commit/f06018002f5fd3d68569e194085d35159bebaebb in order for the `npcTestEnableDevelopmentHardForkEras` to no longer be required by Consensus. (IE the result of PR input-output-hk/ouroboros-network#3295)